### PR TITLE
pyupgrade: Modernize syntax for Python >= 3.7

### PR DESCRIPTION
**Pull request by cclauss**
[`pyupgrade --py37-plus **/*.py`](https://pypi.org/project/pyupgrade)

### CODE REVIEW

File: docs/development/includes.py [hunk 1]
```diff
@@ -153,14 +153,14 @@ def inspect_member(node, name_prefix=''):
...
-            url = '%s/%s#%s' % (doxygen_base, anchorfile, anchor)
+            url = '{}/{}#{}'.format(doxygen_base, anchorfile, anchor)
 
             doxygen[name] = {'url': url}
 
             if node.attrib['kind'] == 'function':
                 ret_type = node.find('type').text
                 arglist = node.find('arglist').text
-                sig = '%s %s%s' % (ret_type, name, arglist)
+                sig = '{} {}{}'.format(ret_type, name, arglist)
```
Comment (jlaine) [inline docs/development/includes.py:163]: This could also be an f-string

**Comment by jlaine** [comment]
I like:
- dropping the "object" base class
- removing obsolete `u".."` markers

The string formatting changes however are quite arbitrary and not even consistent. Sometimes f-string are used, sometimes not..

**Comment by cclauss** [comment]
https://pypi.org/project/flynt will be more aggressive with f-string conversion while pyupgrade explains why it is _intentionally timid_ https://github.com/asottile/pyupgrade#f-strings

**Comment by cclauss** [comment]
Should I run flynt also?

**Comment by jlaine** [comment]
I've cherry-picked the changes I'm comfortable with in #1202

**Pull request closed without merging**